issuetracker/gitlab/gitlab.go

```diff
@@ -0,0 +1,37 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/preslavmihaylov/todocheck/common"
+	"github.com/preslavmihaylov/todocheck/issuetracker"
+)
+
+// IssueTracker is an issue tracker implementation for integrating with public & private gitlab issue trackers
+type IssueTracker struct {
+	Origin string
+}
+
+// TaskModel returns the model representing a deserialized gitlab task
+func (it *IssueTracker) TaskModel() issuetracker.Task {
+	return &Task{}
+}
+
+// TaskURLFrom taskID returns the url for the target gitlab task ID to fetch
+func (it *IssueTracker) TaskURLFrom(taskID string) string {
+	return taskID
+}
+
+// IssueAPIOrigin returns the URL for github's issue-fetching API
+func (it *IssueTracker) IssueAPIOrigin() string {
+	tokens := common.RemoveEmptyTokens(strings.Split(it.Origin, "/"))
+	if tokens[0] != "http:" && tokens[0] != "https:" {
```

Comment:
The use of `HasPrefix` method (from the GitHub implementation) is much cleaner I think.

Reply:
I agree. I didn't know of this function. Similarly to my comment above, would you agree to add this in scope for a follow-up task to avoid making too many changes in this PR?

Reply:
Sure!